### PR TITLE
Updating vip cache commands descriptions and examples to follow the VIP-CLI style guide

### PR DESCRIPTION
**Pull request by terriann**
Updated the voice, tense, and format of descriptions and examples for the `vip cache` commands to follow the VIP-CLI style guide.

## Pull request checklist

- [ n/a ] Update [SETUP.md](https://github.com/Automattic/vip-cli/blob/trunk/docs/SETUP.md#list-of-environmental-variables) with any new environmental variables.
- [ ] Update [the documentation](https://github.com/Automattic/vip-cli/blob/trunk/docs).
- [x] [Manually test](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#manual-testing) the relevant changes.
- [ x ] Follow the [pull request checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#new-pull-requests)
- [ x ] Add/update [automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) as needed.

## New release checklist

- [ ] [Automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) pass.
- [ ] The [Preparing for release checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#preparing-for-release) is completed.

## Steps to Test

Observe the results of the `--help` display for impacted commands, such as:

- `vip --help`
- `vip cache --help`
- `vip cache purge-url --help`

### CODE REVIEW

File: src/bin/vip-cache-purge-url.js [hunk 1]
```diff
@@ -64,8 +70,8 @@ command( {
 	appQuery,
 	envContext: true,
 	wildcardCommand: true,
-	usage: 'vip cache purge-url <URL>',
+	usage,
 } )
-	.option( 'from-file', 'Read URLs from file (useful to purge multiple URLs)' )
+	.option( 'from-file', 'Read URLs from a file, can be used to purge multiple URLs.' )
```
Comment (yolih) [inline src/bin/vip-cache-purge-url.js]: I see that you provided more instructions for the use of  `--from-file` in the command example, but I'm wondering if we could include that in the description instead. If a user is reading this in the help menu for `vip cache purge-url`, they already have the context for using `--from-file` for purging a URL from cache. So maybe the description could simply be:
> Read one or more URLs from a file, each listed on a single line.

Comment (terriann) [reply src/bin/vip-cache-purge-url.js]: Good call! I pushed an update to use that description.

File: src/bin/vip-cache-purge-url.js [hunk 2]
```diff
@@ -6,14 +6,20 @@ import * as exit from '../lib/cli/exit';
...
+const exampleUsage = 'vip @example-app.develop cache purge-url';
+
 const examples = [
 	{
-		usage: 'vip cache purge-url <URL>',
-		description: 'Purge a URL from page cache',
+		usage:
+			`${ exampleUsage } https://example-app-develop.go-vip.co/sample-page/` +
+			'\n    - Purged URL: https://example-app.develop.go-vip.co/sample-page/',
+		description: 'Purge the page cache for a single URL.',
```
Comment (sanmai) [inline src/bin/vip-cache-purge-url.js:17]: Looks like this:
Screenshot

**Review by sanmai — APPROVED** [review]
Tested with:
```
./dist/bin/vip-cache-purge-url.js -h
./dist/bin/vip-cache.js -h
```

File: src/bin/vip-cache-purge-url.js (hunk 2 above)
Comment (yolih) [reply src/bin/vip-cache-purge-url.js:17]: I just tested this and can confirm that the example output matches the actual output when that command is run successfully. 👍

**Comment by yolih** [comment]
LGTM. Thanks for the careful review, @sanmai !

**Pull request merged**